site/en/docs/devtools/console/reference/index.md

```diff
@@ -87,6 +87,38 @@ messages][15] for a hands-on walkthrough.
 [Open Console Settings][16] and disable **Group similar** to disable the Console's default message
 grouping behavior. See [Log XHR and Fetch requests][17] for an example.
 
+### View stack traces {: #view-stack-traces }
+
+The **Console** automatically captures stack traces for errors and warnings. A stack trace is a history of function calls (frames) that led to the error or warning. The **Console** shows them in reverse order: the latest frame is at the top.
+
+To view a stack trace, click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/MaSHbLXzcTFbxhx9K8hX.svg", alt="Expand.", width="24", height="24" %} expand icon next to an error or warning.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/TINzuAQUw7Ug3g2pOljm.png", alt="Stack traces.", width="800", height="401" %}
+
+#### View async stack traces {: #async-stack-traces }
+
+If supported by the framework you are using, DevTools can trace async operations by linking both parts of the async code together.
```

Comment:
"If supported by the framework you are using, or when using browser scheduling primitives directly (such as `setTimeout`), DevTools can trace async operations by linking both parts of the async code together."